keys.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"github.com/jroimartin/gocui"
+)
+
+type Binding struct {
+	Key     interface{} // so both gocui.Key and rune work
+	Mod     gocui.Modifier
+	Handler func(g *gocui.Gui, v *gocui.View) error
+}
+
+func (vc *ViewController) CursorUp(g *gocui.Gui, v *gocui.View) error {
```

Comment:
It's usually a good practice to keep all methods in the same file as a struct type declaration.

Reply:
k